core/templates/dev/head/app.js

```diff
@@ -270,6 +203,186 @@ oppia.config(['$provide', function($provide) {
   ]);
 }]);
 
+// Dynamically generate CKEditor widgets for the rich text components.
+oppia.run([
+  '$timeout', '$compile', '$rootScope', '$uibModal', 'RteHelperService',
+  'HtmlEscaperService', 'taOptions',
+  function($timeout, $compile, $rootScope, $uibModal, RteHelperService,
+      HtmlEscaperService, taOptions) {
+    taOptions.disableSanitizer = true;
```

Comment:
Not sure why you need to touch `taOptions` here? `taOptions` is for setting up textangular, this block of code is for CKEditor.

Reply:
Oh, my bad, I was testing stuff and forgot to remove this part. Fixed this!